feature-libs/estimated-delivery-date/schematics/add-estimated-delivery-date/__snapshots__/index_spec.ts.snap

```diff
@@ -1,134 +1,21 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Spartacus Estimated Delivery Date schematics: ng-add Estimated Delivery Date feature eager loading should import appropriate modules 1`] = `""`;
+exports[`Spartacus Estimated-Delivery-Date schematics: ng-add Estimated-Delivery-Date feature eager loading should import appropriate modules 1`] = `""`;
 
-exports[`Spartacus Estimated Delivery Date schematics: ng-add Estimated Delivery Date feature general setup should add the feature using the lazy loading syntax 1`] = `""`;
+exports[`Spartacus Estimated-Delivery-Date schematics: ng-add Estimated-Delivery-Date feature general setup should add the feature using the lazy loading syntax 1`] = `""`;
 
-exports[`Spartacus Estimated Delivery Date schematics: ng-add Estimated Delivery Date feature general setup styling should create a proper scss file 1`] = `
-"@import "../../styles-config";
-@import "@spartacus/estimated-delivery-date";
-"
-`;
+exports[`Spartacus Estimated-Delivery-Date schematics: ng-add Estimated-Delivery-Date feature general setup should add the feature using the lazy loading syntax 2`] = `
+"import { NgModule } from '@angular/core';
+import { CartBaseModule } from "@spartacus/cart/base";
+import { EstimatedDeliveryDateModule } from "@spartacus/estimated-delivery-date";
 
-exports[`Spartacus Estimated Delivery Date schematics: ng-add Estimated Delivery Date feature general setup styling should update angular.json 1`] = `
```

Comment:
Why was this test removed? does this feature use `estimated-delivery-date.scss`?

Reply:
Removed by mistake added it back